src/database/index.ts

```diff
@@ -791,4 +791,72 @@ export class Database {
 
     return result ?? null;
   }
+
+  async deleteChainData(chainId: ChainId) {
+    this.#logger.info("Deleting chain data for chainId:", chainId);
+
+    await this.#db.transaction().execute(async (trx) => {
```

Comment:
- rename to tx ?
- Also we should likely do `this.#db.withSchema("schemaName").trans...` cause else this will break once we add the IPFS schema

Reply:
@thelostone-mc will update this after your PR was merged, ao  i can rebase and have the new schema names available